src/middlewared/middlewared/plugins/chart_releases_linux/rollback.py

```diff
@@ -31,15 +33,22 @@ async def rollback(self, job, release_name, options):
 
         `item_version` is version which we want to rollback a chart release to.
 
-        `rollback_snapshot` is a boolean value which when set will rollback snapshots of ix_volumes.
+        `rollback_snapshot` is a boolean value which when set will rollback snapshots of any PVC's or ix volumes being
+        consumed by the chart release.
 
-        `force` is a boolean passed to helm for rollback of a chart release and also used for rolling back snapshots
-        of ix_volumes.
+        `force_rollback` is a boolean which when set will force rollback operation to move forward even if no
+        snapshots are found. This is only useful when `rollback_snapshot` is set.
 
-        It should be noted that rollback is not possible if a chart release is using persistent volume claims
-        as they are immutable.
-        Rollback is only functional for the actual configuration of the release at the `item_version` specified and
-        any associated `ix_volumes`.
+        `recreate_resources` is a boolean which will delete and then create the kubernetes resources on rollback
+        of chart release. This should be used with caution as if chart release is consuming immutable objects like
+        a PVC, the rollback operation can't be performed and will fail as helm tries to do a 3 way patch for rollback.
+
+        `force` is a boolean which when set will override values provided for `force_rollback` and `recreate_resources`
```

Comment:
Why are we already having deprecated options in early ALPHA releases?

Reply:
Yes because we should separate rollback/recreate_resources options. Earlier `force` did both which is not desired.

Reply:
Why do we keep it if we have not releases any stable API yet?